content/applications/inventory_and_mrp/inventory/management/misc/cluster_picking.rst

```diff
@@ -0,0 +1,233 @@
+   temporarily placed in a container during picking before getting transferred to their final
+   shipping box.
+
+   Alternatively, use the :guilabel:`Destination Package` field when the product is directly placed
+   in its *disposable* shipping box during picking.
+
+.. example::
+   Process the cluster batch for the three orders of apples, oranges, and bananas :ref:`example
+   <inventory/misc/cluster_picking/example>` by assigning each picking to a dedicated package.
+
+   At the storage location for apples, `WH/Stock/Shelf A`, assign the apples in all three pickings
+   to one of the three disposable packages, `CLUSTER-PACK-1`, `CLUSTER-PACK-2`, and
+   `CLUSTER-PACK-3`.
+
+   Record this in Odoo using the :guilabel:`Destination Package` field in the
+   :guilabel:`Detailed Operations` tab.
+
+   .. image:: cluster_picking/cluster-batch-example.png
+      :align: center
+      :alt: Example of processing cluster pickings in *Inventory*.
+
+In Barcode
+----------
+
+To process cluster pickings directly from *Barcode*, select the :guilabel:`Batch Transfers` button
+from the *Barcode* dashboard. Then, select the desired card for the batch.
+
+On the batch transfer screen, the products in the picking are grouped by location, and each line is
+color-coded to associate products in the same picking together.
+
+Then, follow the prompt to :guilabel:`Scan the source location` barcode for the storage location the
+first product is located at. Then, scan the barcode for the product and package to process the
+transfer. Repeat this for all products and click the :guilabel:`Validate` button.
+
+.. note::
+   To find the package barcode, navigate to :menuselection:`Inventory app --> Products -->
+   Packages`, select the desired package, click the :guilabel:`⚙️ (gear)` icon at the top of the
+   package form, and select the :guilabel:`Print` option. Next, select one of the three print
+   options to generate the package barcode from the :guilabel:`Package Reference` field.
+
+   .. image:: cluster_picking/find-package-barcode.png
+      :align: center
+      :alt: Display where the package barcode can be generated.
+
+.. example::
+   Begin processing the cluster picking by going to the first storage location, `Shelf A`, and
+   scanning the :ref:`location barcode <barcode/setup/location>`. Doing so highlights all the
+   pickings that need products from this particular location.
+
+   Scan the barcode for the apple, which highlights the picking labeled in red for the product
+   `Apple`, for the picking, `WH/PICK/00007`.
+
+   Then, scan the `CLUSTER-PACK-1` package barcode, and place the product in the designated package.
+
+   .. image:: cluster_picking/batch-barcode.png
+      :align: center
+      :alt: Example of cluster batch from the *Barcode* app.
+
+.. tip::
+   After assigning a package to a picking, Odoo suggests scanning the same package by displaying the
+   package name under another product related to the picking.
```

Comment:
Sorry, I don't fully understand this. What does this do / why does Odoo suggest it?

Reply:
Added a bit more context:
> After creating a batch transfer and assigning a package to a picking, Odoo suggests the specified
   package by displaying the name *in italics* under the product name, ensuring pickers place
   products into the correct boxes.